deploy/packer/app.json

```diff
@@ -26,6 +26,30 @@
       "destination":    "/tmp/deploy.sh"
     },
     {
+      "type":           "file",
+      "source":         "deploy/bin/gen-signing-key",
+      "destination":    "/tmp/gen-signing-key"
+    },
+    {
+      "type":           "shell",
+      "inline":         ["chmod u+x /tmp/gen-signing-key"]
+    },
+    {
+      "type":           "shell",
+      "inline":         ["/tmp/gen-signing-key"],
+      "only":           ["hvm-builder", "pv-builder"]
+    },
+    {
+      "type":           "file",
+      "source":         "{{user `cotSigningKey`}}",
```

Comment:
Are generating the private key on our host/master machine and then copying it to the AMI?

Note: I think it would be slightly safer to generate the key on the AMI and copy out the public key, that way it ever only exists on the worker AMI, the private key never gets into memory or disk of the host/master machine...

Granted this might be a follow up someday when we have time :)

Reply:
We're using the same private/public keypair on all AMIs.